cadasta/test/resource_tests/test_attaching.py

```diff
@@ -566,6 +851,66 @@ def test_file_with_nonacceptable_mime_type_cannot_be_uploaded(
             '//*[contains(@class, "file-well")]'
             '//*[normalize-space()="File type not allowed."]')
 
+    @pytest.mark.uploads
+    def test_multiple_existing_resources_from_different_pages_can_be_attached(
```

Comment:
This test fails with `selenium.common.exceptions.WebDriverException: Message: Failed to decode response from marionette`.

Following that there's another error during tear down (`ERROR at teardown of TestAttaching.test_multiple_existing_resources_from_different_pages_can_be_attached`): `selenium.common.exceptions.SessionNotCreatedException: Message: Tried to run command without establishing a connection`

Reply:
This can be solved by increasing the VM's RAM to 3GB. When I'm running the test with the default 2GB, I consistently get the `WebDriverException` error. This goes away when I increased the size to 3GB. I noticed this because the test didn't fail when run on Travis and this is the longest test we have (so far) because of the need to set up and then clean up 11 resource files, which implies that the test needs more computing resources. I have no idea why there is no out-of-memory error message or something that appears.

Reply:
Ask discussed in our call, try to use fixtures instead of uploading all the files.

Reply:
I've add a commit to the branch to remove the 11 dummy resources and replace them with 11 test fixture objects of model `resources.resource` and updated the test to match.